src/index.js

```diff
@@ -22,12 +24,17 @@ export default class SpoorClient {
 		this.category = category;
 		this.product = product;
 		this.req = req;
+		this.cookies = cookies;
+		this.ua = ua;
 		this.shouldSubmitEvent = submitIf;
 		this.apiKey = apiKey;
 		this.inTestMode = inTestMode;
 	}
 
-	submit ({source=this.source, category=this.category, req=this.req, apiKey=this.apiKey, product=this.product, action, context}={}) {
+	submit ({source=this.source, category=this.category, req=this.req, cookies=this.cookies, ua=this.ua, apiKey=this.apiKey, product=this.product, action, context}={}) {
```

Comment:
These parameters are getting a bit unwieldy. Not now but soon we should split it over several lines as in `constructor` (or find a nicer way of doing this).